automated/linux/iperf/iperf.sh

```diff
@@ -15,17 +15,23 @@ TIME="10"
 THREADS="1"
 # Specify iperf3 version for CentOS.
 VERSION="3.1.4"
+# By default, the client sends to the server,
+# Setting REVERSE to -R means the server sends to the client
+REVERSE=""
+AFFINITY="-A 0"
 
 usage() {
-    echo "Usage: $0 [-c server] [-t time] [-p number] [-v version] [-s true|false]" 1>&2
+    echo "Usage: $0 [-c server] [ -e server eth device] [-t time] [-p number] [-v version] [-R] [-s true|false]" 1>&2
     exit 1
 }
 
-while getopts "c:t:p:v:s:h" o; do
+while getopts "A:c:t:p:v:s:Rh" o; do
```

Comment:
-e is missing

Reply:
I dropped -e and didn't update correctly. I realised iperf-server.yaml is the only one that needs to know the server's ethernet address, since we keep the lava-* commands in the yaml only.